ui/src/views/compute/DeployVM.vue

```diff
@@ -1133,6 +1133,10 @@ export default {
         this.vm.disksizetotalgb = this.diskSize
       }
 
+      if (this.serviceOffering.rootdisksize) {
+        this.vm.disksizetotalgb = this.serviceOffering.rootdisksize
+      }
```

Comment:
@GabrielBrascher just a suggestion

```
       if (this.serviceOffering.rootdisksize) {
         this.vm.disksizetotalgb = this.serviceOffering.rootdisksize
       } else if (this.diskSize) {
         this.vm.disksizetotalgb = this.diskSize
       }
```

Reply:
Your suggestion looks better indeed, I will add it into the PR.
Thanks for the review, @sureshanaparti.